src/scripts/create-config.js

```diff
@@ -80,14 +84,26 @@ const run = async () => {
         });
     }
 
-    if (argv.service === "all") {
-        for (const cfgKey of Object.keys(cfg)) {
-            await createConfig(cfgKey, cfg[cfgKey]);
+    // Expand services
+    if (argv.services.length === 1 && argv.services[0] === "all") {
+        argv.services = Object.keys(cfg);
+    }
+
+    for (const service of argv.services) {
+        if (!(service in cfg)) {
+            console.error(`Cannot find service ${service} in config`);
+
+            return;
         }
-    } else if (argv.service in cfg) {
-        await createConfig(argv.service, cfg[argv.service]);
-    } else {
-        console.error(`Cannot find service ${argv.service} in config`);
+    }
+
+    for (const service of argv.services) {
+        if (argv.loglevel) {
+            cfg[service] = Object.assign({}, cfg[service], {
+                level: argv.loglevel });
```

Comment:
Strange code-style... Would rather like to see this style which is more "standard"...
```js
cfg[service] = Object.assign({}, cfg[service], {
    level: argv.loglevel
});
```